docs/admin_guide.md

```diff
@@ -280,31 +307,64 @@ Description:               Run Compaction for given instant time
 * compaction run - Run Compaction for given instant time
 ```
 
-##### Up-Coming CLI for Compaction
-
-In the next release, more useful CLI to revert/repair compaction schedules will be added. Here is a preview of them:
+##### Validate Compaction
 
 Validating a compaction plan : Check if all the files necessary for compactions are present and are valid
 
 ```
-hoodie:trips->compaction validate --compactionInstant <instantId>
+hoodie:stock_ticks_mor->compaction validate --instant 20181005222611
+...
+
+   COMPACTION PLAN VALID
+   
+    ___________________________________________________________________________________________________________________________________________________________________________________________________________________________
+    | File Id                             | Base Instant Time| Base Data File                                                                                                                   | Num Delta Files| Valid| Error|
+    |==========================================================================================================================================================================================================================|
+    | 05320e98-9a57-4c38-b809-a6beaaeb36bd| 20181005222445   | hdfs://namenode:8020/user/hive/warehouse/stock_ticks_mor/2018/08/31/05320e98-9a57-4c38-b809-a6beaaeb36bd_0_20181005222445.parquet| 1              | true |      |
+
 ```
 
+##### NOTE
+
 The following commands must be executed without any other writer/ingestion application running.
 
 Sometimes, it becomes necessary to remove a fileId from a compaction-plan inorder to speed-up or unblock compaction 
 operation. Any new log-files that happened on this file after the compaction got scheduled will be safely renamed 
 so that are preserved. Hudi provides the following CLI to support it
 
+
+##### UnScheduling Compaction
+
 ```
 hoodie:trips->compaction unscheduleFileId --fileId <FileUUID>
+....
+No File renames needed to unschedule compaction. Compaction successfully unscheduled
```

Comment:
Is this text correct ? Here if we unschedule a fileId, may be we should say 👍 

No File renames needed to unschedule file id from compaction. FileId successfully unscheduled from compaction plan.. ?

Reply:
It was coming from a reused method. Customized the message.